includes/Plugin_Main.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Plugin_Main
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check;
+
+/**
+ * Main class for the plugin
```

Comment:
```suggestion
 * Main class for the plugin.
```
As per [Inline document CS](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#2-classes), Use a period at the end.